src/components/InlineWrapper/InlineWrapper.story.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import InlineWrapper from './InlineWrapper';
+import docs from './InlineWrapper.docs.mdx';
+
+export default {
+	title: 'InlineWrapper',
+	component: InlineWrapper,
+	parameters: {
+		docs: { page: docs },
+	},
+};
+
+const styles = {
+	width: 400,
+	margin: '0 auto',
+};
+
+export const Default = () => (
+	<div style={styles}>
+		<InlineWrapper>
+			<p>An <strong>inline wrapper</strong> encapsulates components that add context or additional functionality to a page.</p>
```

Comment:
Could you use some 'real' content here that helps others understand when to use this component? Another example or two would be helpful too

Reply:
I added some more realistic content in. What did you have in mind re: other examples? `ArticleCallout` shows dark mode as an example, and we use `ArticleCallout` with forms, but we don't have the `ColorScheme` component or any of our form components in this repo.